src/main/resources/application-release.properties

```diff
@@ -1,7 +1,10 @@
 # Database credentials
-spring.datasource.url=${CLEARDB_DATABASE_URL}
-spring.datasource.username=${CLEARDB_DATABASE_USERNAME}
-spring.datasource.password=${CLEARDB_DATABASE_PASSWORD}
+spring.datasource.url=jdbc:${AWS_DATABASE_URL}
+spring.datasource.username=${AWS_DATABASE_USERNAME}
+spring.datasource.password=${AWS_DATABASE_PASSWORD}
+
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
```

Comment:
should we really specify driver and dialect? I think it can be found automatically from the url